### ISSUE TITLE
feat: slack bot example - any plans?

### ISSUE THREAD

**Issue by matgren** [callstackincubator/fabrice-ai#88]
Awesome work guys! just checking so I don't do same stuff - are there any plans to ship some fullstack examples f.e. slack bot ?

**Comment by pkarw** [comment]
Thanks 🙏 definitely it's something we're looking for however due to limited capabilities at the moment we probably won't be working on this case in the upcoming days 

So! Any help is greatly appreciated and maybe this is the next idea for your contribution? 😁

I think you could create a separate GitHub project just for this example - surely it's gonna be hot. This is one of three cases we had in mind implementing the server features. Please check the 'medical_survey_server' example

**Comment by grabbou** [comment]
Something I've been thinking about thanks to our async architecture

- on new email, send slack message, wait for response (whenever it arrives), respond to email

That could be pretty cool! Happy to help if things break in the framework down the way 😂

**Comment by pkarw** [comment]
Yeah! @grabbou, these examples are more than welcome. @matgren happy to support you :)

**Comment by matgren** [comment]
awesome! thank you for the prompt response. I think making separate Github example is the way to go. I will try to carve some time to do it. Will keep you posted.

**Comment by pkarw** [comment]
Awesome! Thanks!

**Comment by pkarw** [comment]
@matgren how things are going on your end? :)
